weather/contrib/ecowitt/server.py

```diff
@@ -275,23 +289,32 @@ def fetchWeather():
                     if INFLUX:
                         log.debug("Writing to InfluxDB")
                         try:
-                            client = InfluxDBClient(host=IHOST,
-                                port=IPORT,
-                                username=IUSER,
-                                password=IPASS,
-                                database=IDB)
+                            if ITOKEN == "":
+                                # Influx 1.8
+                                client = InfluxDBClient(
+                                    url="http://%s:%s" % (IHOST,IPORT),
+                                    token="%s:%s" % (IUSER,IPASS),
```

Comment:
That's it!  It only impacts 1.8 installs if you turn auth ON and set a user/pass, otherwise InfluxDB 1.8  doesn't care what you send it.